src/_guides/language/language-tour.md

```diff
@@ -2591,21 +2592,14 @@ Besides invoking a superclass constructor, you can also initialize
 instance variables before the constructor body runs. Separate
 initializers with commas.
 
-<?code-excerpt "language-tour/reference/initializer_list.dart"?>
+<?code-excerpt "lib/language_tour/classes/point_alt.dart (initializer-list)"?>
 {% prettify dart %}
-class Point {
-  num x;
-  num y;
-
-  Point(this.x, this.y);
-
-  // Initializer list sets instance variables before
-  // the constructor body runs.
-  Point.fromJson(Map json)
-      : x = json['x'],
-        y = json['y'] {
-    print('In Point.fromJson(): ($x, $y)');
-  }
+// Initializer list sets instance variables before
+// the constructor body runs.
+Point.fromJson(Map json)
+    : x = json['x'],
+      y = json['y'] {
+  print('In Point.fromJson(): ($x, $y)');
```

Comment:
I trimmed the code excerpt to show only the constructor being discussed (since the full class is shown repeated times earlier).